model/src/main/proto/platform_parameter.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// It will correspond to a single Platform Parameter Value.
+message PlatformParameter {
+    // Value of a parameter can be of any one of the following type.
+    oneof value {
+        // Boolean type parameter value.
+        bool boolean = 2;
+        // Integer type parameter value.
+        int32 integer = 3;
+        // String type parameter value.
+        string string = 4;
+    }
+}
+
+// Storage format of platform parameters while the app is running.
+message PlatformParameterMap{
+    map<string, PlatformParameter> parameter = 1;
+}
+
+// Pair of Platform Parameter Value and Name.
+message PlatformParameterWithName{
+    string name = 1;
+    PlatformParameter value = 2;
+}
+
+// Final storage format of parameters in the disk. This is made in the form of list so as to closely
+// resemble the JSON response in cache.
+message PlatformParameterDatabase {
+    // List of Parameters values with their names.
```

Comment:
Or maybe: ``Platform parameters cached from the backend.`` might be clearer on the context.

Given that, it might also be worthwhile to name the overall structure ``RemotePlatformParameterDatabase`` since we actually have 2 types of parameters (local & remote), and it seems this is only storing the latter--is that correct?

Reply:
That's True that it is stoting the Remote Parameters. Done